b3lb/rest/endpoints/backend/meeting.py

```diff
@@ -30,16 +30,24 @@ def backend_end_meeting_callback(request):
     Custom callback URL for end meeting.
     """
     parameters = request.GET
-    print(parameters)
-    if "nonce" not in parameters:
-        return HttpResponse("Unauthorized", status=401)
-
+    if "end_nonce" not in parameters:
+        return HttpResponse(status=204)
     try:
-        meeting = Meeting.objects.get(end_nonce=parameters["nonce"])
-        requests.get(meeting.end_callback_url)
-        meeting.is_running = False
-        meeting.save()
+        meeting = Meeting.objects.get(id=parameters["meetingID"], end_nonce=parameters["end_nonce"])
+        if meeting.end_callback_url:
+            url_postfix = "meetingID={}&recordingmarks={}".format(parameters["meetingID"], parameters["recordingmarks"])
```

Comment:
Looks like using untrusted input without escaping.

Reply:
recordmarkins is now checked before used